website/src/components/Chat/PluginsChooser.tsx

```diff
@@ -0,0 +1,257 @@
+import {
+  Button,
+  Text,
+  Tooltip,
+  Radio,
+  RadioGroup,
+  Box,
+  Editable,
+  EditableInput,
+  EditablePreview,
+  Image,
+  FormControl,
+  FormLabel,
+  Icon,
+  IconButton,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Stack,
+  Textarea,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { Cog, Edit, Plus } from "lucide-react";
+// i need
+import { AttachmentIcon, WarningIcon, CheckCircleIcon, CloseIcon } from "@chakra-ui/icons";
+import { ChangeEvent, useCallback, useEffect, useState, useRef } from "react";
+import { useTranslation } from "next-i18next";
+import { Controller, useFormContext } from "react-hook-form";
+import { ChatConfigFormData } from "src/types/Chat";
+import { useChatContext } from "./ChatContext";
+import { PluginEntry } from "src/types/Chat";
+import { get, post } from "src/lib/api";
+import { API_ROUTES } from "src/lib/routes";
+
+// TODO: Delete me!
```

Comment:
please do!

Reply:
Can you help me solve width issues, I didn't come up with solution?

Reply:
If you remember we tried it together once on discord call, but I just left it like that